org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/DebugAdapter.java

```diff
@@ -22,7 +22,6 @@
 import java.util.function.Consumer;
 
 import org.eclipse.core.runtime.CoreException;
```

Comment:
Please don't use eclipse exception type in adapter. In my PR #29, i have defined a new DebugException type, we could use that type to wrap eclipse exception. 
BTW, we can fix exception type in a new change list.